fw/rbcx-coprocessor/include/utils/TaskWrapper.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <functional>
+
+#include "FreeRTOS.h"
+#include "task.h"
+
+template <int StackSizeBytes> class TaskWrapper {
+public:
+    TaskWrapper()
+        : m_handle(nullptr) {}
+
+    ~TaskWrapper() { abort(); }
+
+    bool start(
+        const char* name, UBaseType_t priority, std::function<void()>&& func) {
```

Comment:
*function* myslím uvnitř něco alokuje, odkud se ta paměť bere?

Reply:
Já právě nevím! Taky jsem si to myslel, ale malloc to nepotřebuje (poznám podle využití flash/ram) a funguje to. Možná je alokace potřeba pouze pro nějaké komplikovanější casy? Tady to ještě není, ale funguje mi to i s capture `[this]() { ... }`

Reply:
Není potřeba se zalamovat - task jede